LEGO1/omni/include/mxbitset.h

```diff
@@ -13,6 +13,7 @@ class MxBitset {
 public:
 	MxBitset() { Tidy(); }
 
+	// SIZE 0x8
```

Comment:
0x08

Reply:
haha

Reply:
My next script should be the tool that catches all these things